msu/systems/debug/debug_system.nut

```diff
@@ -119,4 +119,10 @@
 			}
 		}
 	}
+
+	function addPopupMessage(_info)
+	{
+		::MSU.Popup.addMessage(_info);
+		::MSU.Popup.setInfoText(_info.modID);
```

Comment:
What's the difference between the message and the info text and where is each shown in the UI? I assume the message is the entry in the popup itself.

Reply:
Info text is below the item
![image](https://github.com/MSUTeam/MSU/assets/34767248/4dd1545d-67f1-43b8-9afe-21011924a95a)

message is in the popup
![image](https://github.com/MSUTeam/MSU/assets/34767248/aab17252-9cb4-4243-b839-bba62f859f92)